pkg/releasesync/releasesync_test.go

```diff
@@ -166,3 +170,119 @@ func buildReleaseDetails(hasFailedBlockingJobs bool) ReleaseDetails {
 	return releaseDetails
 
 }
+
+func TestChangeLog(t *testing.T) {
```

Comment:
Thanks for the tests. I do find it a bit cumbersome to read, IMHO it should not be 100+ lines.  I'd maybe see about reducing some of the lines that were duplicated from releasesync.go into common helpers